CHANGELOG.md

```diff
@@ -2,464 +2,311 @@
 
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+23+Ending+May+11%2C+2015%22+label%3Aenhancement) 
 
-## Unreleased - as of Sprint 21 end 2015-03-30
+- Providers: Better handling of power states
+  - OpenStack Cloud (paused, rebooting, waiting to launch)
+  - Amazon (powering up/down)
+- UI: OpenStack Terminology
+  - Clusters vs Deployment Roles
+  - Hosts vs Nodes
+- UI: Foreman Enhancements
+- Appliance: Lazy require the less-rails gem
 
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+21+Ending+Mar+30%2C+2015%22+label%3Aenhancement)
+### [Fixed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+23+Ending+May+11%2C+2015%22+label%3Abug)
 
-- Providers: OpenStack
+Notable fixes include:
+- Clean up of CPU details in UI
+- Chargeback
+  - Storage C&U collected every 60 minutes.
+  - Don’t collect cpus/memory available unless you have usage.
+- Implemented  workaround for broker hang
+  - Reported as VMware events and capacity and utilization works for a while, then stops.
+
+# Botvinnik Release
+
+### Added Features
+#### Providers
+- General
+  - Added refresh status and errors, viewable on Provider Summary Page.
+  - Added collection of raw power state and exposed to reporting.
+  - Orchestration: Support for Retirement
+  - Update authentication status when clicking validate for existing Providers
+  - Hosts and Providers now default to use the hostname column for communication instead of IP address.    
+- SCVMM
+  - Provisioning from template, Kerberos Authentication
+  - Virtual DVD drives for templates
+- Containers: UI updates, refresh button
+- Foreman
+  - Added tag processing during provisioning
+  - Added inventory collection of direct and inherited host/host-group settings
+- Kubernetes 
+  - Inventory Collection
+  - EMS Refresh scheduling
+- The Foreman
+  - Provider refresh
+  - Enabled Reporting / Tagging
+  - Automate service models
+  - Zone enablement
+  - EMS Refresh scheduling
+- Cloud Providers
+ - Cloud Stacks: Summary and list views.
+ - Cloud images and instances: Added root device type to summary screens.
+ - Cloud flavors: Added block storage restriction to summary screens.
+ - Reporting
+- OpenStack
   - Inventory for Heat Stacks (Cloud and Infrastructure)
   - Connect Cloud provider to Infrastructure provider
-- Providers: General
-  - EMS Refresh scheduling for Foreman and Kubernetes
-- Rest API
-  - Providers CRUD
-  - Refresh via /api/providers
-- Provisioning
-  - Heat Orchestration provisioning through services
-  - Foreman
-     - Provisioning of bare metal systems
-     - Organization and location inventory
-     - Uses latest Foreman Apipie gem
-- Automate: Enhanced UI import
-- Fleecing
- - Qcow3 support
- - VSAN support (VMware)
- - OpenStack instances
-- UI
- - Foreman includes provisioning
+  - OpenStack Infrastructure Host Events
+  - Autoscale compute nodes
```

Comment:
What I'm saying is that I think this should be worded "Autoscale computer nodes supported via automate."